presto-main/src/main/java/com/facebook/presto/operator/aggregation/builder/InMemoryHashAggregationBuilder.java

```diff
@@ -171,9 +170,14 @@ public double getExpectedHashCollisions()
     }
 
     @Override
-    public ListenableFuture<?> isBlocked()
+    public ListenableFuture<?> startMemoryRevoke()
+    {
+        return null;
```

Comment:
I will throw `UnsupportedOperationException` from `startMemoryRevoke` and `finishMemoryRevoke`